src/niscope/metadata/attributes_addon.py

```diff
@@ -156,6 +156,34 @@
     1150345: { "codegen_method": "no" },  # P2P_FIFO_ENDPOINT_COUNT - P2P Attribute
     1150354: { "codegen_method": "no" },  # P2P_ONBOARD_MEMORY_ENABLED - P2P Attribute
     1150380: { "codegen_method": "no" },  # SAMPLES_TRANSFERRED_PER_RECORD - P2P Attribute
+    1150373: { "codegen_method": "no" },  # STREAM_RELATIVE_TO - P2P Attribute - #825
+    1150105: { "codegen_method": "no" },  # OSCILLATOR_PHASE_DAC_VALUE - NI-TClk Attribute - #825
+    1151002: { "codegen_method": "no" },  # MUX_MODE_REGISTER - Internal Attribute - #825
+    1150303: { "codegen_method": "no" },  # DDC_CENTER_FREQUENCY - Onboard Processing Attribute - #823
```

Comment:
I believe all these "Onboard Processing" should be "Onboard Signal Processing".
(feature is marketed as OSP)